loadgen/README.md

```diff
@@ -1,14 +1,11 @@
 # Overview {#mainpage}
 
-*Note:* A compiled html version of this document is hosted online
-[here](https://mlperf.github.io/inference/loadgen/index.html).
-
 ## Introduction
 
 * The LoadGen is a *reusable* module that *efficiently* and *fairly* measures
```

Comment:
Please use camelCase "LoadGen" in all places in this doc.

Reply:
Agree. I will replace them by "LoadGen"